var/spack/repos/builtin/packages/ghostscript/package.py

```diff
@@ -29,14 +29,23 @@ class Ghostscript(Package):
     """An interpreter for the PostScript language and for PDF."""
 
     homepage = "http://ghostscript.com/"
-    url = "http://downloads.ghostscript.com/public/old-gs-releases/ghostscript-9.18.tar.gz"
```

Comment:
Can you still keep the most recent url up here? It's used for things like `spack versions` and `spack checksum`.